src/include/rbd/librbd.h

```diff
@@ -641,6 +641,11 @@ CEPH_RBD_API int rbd_mirror_image_get_status(rbd_image_t image,
                                              rbd_mirror_image_status_t *mirror_image_status,
                                              size_t status_size);
 
+// RBD consistency groups support functions
+CEPH_RBD_API int rbd_create_cg(rados_ioctx_t p, const char *name);
```

Comment:
This was my concern as well. Do you want to change only c bindings or across all the changes?